Stripe/StripeiOSTests/PaymentSheet+LPMTests.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  PaymentSheet+LPMTests.swift
+//  StripeiOSTests
+//
+//  Created by Yuki Tokuhiro on 7/18/23.
+//
+
+import StripeCoreTestUtils
+import XCTest
+
+@testable@_spi(STP) import Stripe
+@testable@_spi(STP) import StripeCore
+@testable@_spi(STP) import StripePayments
+@testable@_spi(STP) import StripePaymentSheet
+@testable@_spi(STP) import StripeUICore
+
+final class PaymentSheet_LPMTests: XCTestCase {
```

Comment:
Is there somewhere i can read up on how these tests work?

Reply:
No, although I plan to add some guidance in this file.

I'm not sure why this is in the diff, though, I think this must have been from a branch that I based this PR off of.